NEWS.md

```diff
@@ -1,13 +1,19 @@
 ## rtables 0.6.9.9002
+### Features
+ * Modified `reorder_split_levels()` to cover more edge cases and be more stringent in the allowed inputs.
+
 ### Bug Fixes
  * Fixed `"\n"` newline issues in `as_html` by relying onto output devices for newline handling. Added `expand_newlines = FALSE` default to allow previous behavior.
+ * `keep_split_levels` throws now an error if the user requests to keep levels that are not present in data.
```

Comment:
This may raise issues downstream. I think it is legitimate, as you can inadvertently create an empty table